app/models/assignment.rb

```diff
@@ -570,31 +574,28 @@ def self.export(csv, parent_id, options)
         names_of_participants += '; ' unless p == team[:team].participants.last
       end
       tcsv << names_of_participants
-
+      export_fields(csv, scores,parent_id, options)
+      csv << tcsv
+    end
+  end
+  def self.export_fields (csv, scores,parent_id, options)    
```

Comment:
There are two `self.export_fields` methods in this file, I am not sure whether method overload works or not. So just rename it.

Reply:
Why you made all newly-added methods class methods?